dask/values.yaml

```
@@ -56,6 +57,7 @@ worker:
     pullSecrets:  # Container image [pull secrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
     #  - name: regcred
   replicas: 3  # Number of workers.
+  #custom_scheduler_url: "" # connect to already existing scheduler, deployed not by this chart
```

```suggestion
  custom_scheduler_url: nil # connect to already existing scheduler, deployed not by this chart
```

I think setting this to `nil` would work the same way and be more readable than having it commented out.

---

updated